src/Uno.UI/Mock/ViewExtensions.netstdref.cs

```diff
@@ -18,9 +18,9 @@ public static partial class ViewExtensions
 
 		internal static FrameworkElement GetTopLevelParent(this UIElement view) => throw new NotImplementedException();
 
-		internal static T FindFirstChild<T>(this FrameworkElement root) where T : FrameworkElement
+		internal static T FindFirstChild<T>(this FrameworkElement root, bool includeCurrent) where T : FrameworkElement
 		{
-			return root.GetDescendants().OfType<T>().FirstOrDefault();
+			throw new NotImplementedException();
```

Comment:
Why did you remove the implementation?

Reply:
Because it's netstdref and anywhere it's calling `GetDescendants()` that will throw this (right after in the code).

Reply:
Ok for the "netstdref", otherwise I would say that the fact `GetDescendants()` is not implemented is not a valid reason. If we gonna implement it, there was about 101% of chances that we forgot to re-implement this method.